charts/rancher-gke-operator/1.1.100/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.6.0-alpha1
```

Comment:
Oh wait, actually can you set max for last chart in questions.yaml to 2.5.99

Reply:
done